python/pyspark/serializers.py

```diff
@@ -187,9 +187,15 @@ def loads(self, obj):
 
 class ArrowStreamSerializer(Serializer):
     """
-    Serializes Arrow record batches as a stream.
+    Serializes Arrow record batches as a stream. Optionally load the ordering of the batches as a
```

Comment:
This is optional. Do we have other usage of this `ArrowStreamSerializer` without the ordering?

Reply:
Yeah, it's also used in the `createDataFrame` path, but that does only use `dump_stream`. Still, it seemed best to make this an optional feature of the serializer.